tests/unit/includes/gateway/bootstrap-test-setup.php

```diff
@@ -18,6 +18,14 @@ function add_action() {}
         }
     }
 
+    /**
```

Comment:
Why this change?

Reply:
We need to close Mockery after tests to clean up resources which we missed to include before.

https://docs.mockery.io/en/latest/reference/expectations.html#expectation-declarations

Reply:
It looks like the function was just moved from one place to another.

Reply:
It's because multiple classes are using Mockery. If a class uses Mockery then we have to close it with `tearDown`. Some classes does not have it because it's parent class includes it. `Bootstrap_Test_Setup` is a parent class to others. I will check the child classes and remove if it has been called unnecessarily.